pkg/ibmcsidriver/ibm_csi_driver.go

```diff
@@ -97,7 +97,7 @@ func (icDriver *IBMCSIDriver) SetupIBMCSIDriver(provider cloudProvider.CloudProv
 	ns := []csi.NodeServiceCapability_RPC_Type{
 		//csi.NodeServiceCapability_RPC_STAGE_UNSTAGE_VOLUME,
 		csi.NodeServiceCapability_RPC_GET_VOLUME_STATS,
-		csi.NodeServiceCapability_RPC_EXPAND_VOLUME,
+		//csi.NodeServiceCapability_RPC_EXPAND_VOLUME,
```

Comment:
please remove this commented line if not needed.

Reply:
@skorati I kept it for future reference purpose. Just like we have kept 
```
//csi.NodeServiceCapability_RPC_STAGE_UNSTAGE_VOLUME
```
commented as well.